tests/test_relative_free_energy.py

```diff
@@ -391,3 +415,4 @@ def test_estimate_free_energy_bisection_invalid_args():
     # toggling the pytest marker
     test_run_hif2a_test_system(estimate_relative_free_energy)
     test_run_hif2a_test_system(estimate_relative_free_energy_bisection)
+    test_run_hif2a_test_system(estimate_relative_free_energy_bisection_hrex)
```

Comment:
Do we still want this since we know it will fail? Could be confusing in the future